src/instrumentsscene/view/instrumentspaneltreemodel.cpp

```diff
@@ -558,6 +564,11 @@ bool InstrumentsPanelTreeModel::isEmpty() const
     return m_rootItem ? m_rootItem->isEmpty() : true;
 }
 
+QString InstrumentsPanelTreeModel::addInstrumentsKeyboardShortcut() const
+{
+    return shortcuts::sequencesToNativeText({ shortcutsRegister()->shortcut(ADD_INSTRUMENTS_ACTIONCODE).sequences[0] });
```

Comment:
this code will cause a crash if the sequence list is empty

Reply:
Thanks, fixed!